modules/heat_conduction/test/tests/gap_heat_transfer_mortar/gap_heat_transfer_sphere3D_mortar.i

```diff
@@ -0,0 +1,212 @@
+sphere_outer_htc = 10 # W/m^2/K
+sphere_outer_Tinf = 300 # K
+
+[GlobalParams]
+  order = SECOND
+  family = LAGRANGE
+[]
+
+[Problem]
+  kernel_coverage_check = false
+  material_coverage_check = false
+[]
+
+[Mesh]
+  [file]
+    type = FileMeshGenerator
+    file = sphere3D.e
+  []
+  [secondary]
+    type = LowerDBlockFromSidesetGenerator
+    sidesets = '2'
+    new_block_id = 10001
+    new_block_name = 'secondary_lower'
+    input = file
+  []
+  [primary]
+    type = LowerDBlockFromSidesetGenerator
+    sidesets = '3'
+    new_block_id = 10000
+    new_block_name = 'primary_lower'
+    input = secondary
+  []
+[]
+
+[Functions]
+  [temp]
+    type = PiecewiseLinear
+    x = '0   1'
+    y = '100 200'
+  []
+[]
+
+[Variables]
+  [temp]
+    initial_condition = 500
+  []
+  [lm]
+    order = FIRST
+    family = LAGRANGE
+    block = 'secondary_lower'
```

Comment:
For this problem, we can use second-order LMs since we have a second-order mesh. We can add more cases in the future.